docker-compose.yml

```diff
@@ -33,24 +36,20 @@ services:
     volumes:
       - esdata:/usr/share/elasticsearch/data
     healthcheck:
-      test: ["CMD-SHELL", "curl -fsSL http://localhost:9200/_cat/health || exit 1"]
+      test: [ "CMD-SHELL", "curl -fsSL http://localhost:9200/_cat/health || exit 1" ]
       interval: 30s
       timeout: 10s
       retries: 3
 
   llama:
+    container_name: fasten-llama
     image: ghcr.io/ggerganov/llama.cpp:server
     volumes:
-      - ./model:/models
+      - ./models:/models
     command: >
-      -m models/Meta-Llama-3-8B-Instruct.Q5_K_M.gguf
-      -n -1
-      -c 512
-      -t 10
-      --host 0.0.0.0
-      --port 8080
+      -m models/Meta-Llama-3.1-8B-Instruct-F16-Q5_K_M.gguf -n 450 -c 2048 -t 10 --host 0.0.0.0 --port 8080
```

Comment:
`-t 10` should be documented, and I am not sure it is really beneficial without parallel execution. Have you tested if there is a latency difference with t=4 for example?

Reply:
I added the -t explanation to documentation in the main README.md. Also specified that the parallel parameter -np is optional and also added the explanation of how to use it